src/main/scala/org/broadinstitute/dsde/firecloud/EntityService.scala

```diff
@@ -374,11 +374,18 @@ class EntityService(rawlsDAO: RawlsDAO, importServiceDAO: ImportServiceDAO, goog
   }
 
   def listJobs(workspaceNamespace: String, workspaceName: String, runningOnly: Boolean, userInfo: UserInfo): Future[List[ImportServiceListResponse]] = {
-    // get jobs from Import Service
-    val importServiceJobs = importServiceDAO.listJobs(workspaceNamespace, workspaceName, runningOnly)(userInfo)
-    // TODO AJ-1602: get jobs from cWDS
-    // TODO AJ-1602: merge lists before replying
-    importServiceJobs
+
+    for {
+      // translate workspace namespace/name to workspaceid
+      workspace <- rawlsDAO.getWorkspace(workspaceNamespace, workspaceName)(userInfo)
+      // get jobs from Import Service
+      importServiceJobs <- importServiceDAO.listJobs(workspaceNamespace, workspaceName, runningOnly)(userInfo)
+      // get jobs from cWDS
+      cwdsJobs = cwdsDAO.listJobsV1(workspace.workspace.workspaceId, runningOnly)(userInfo)
+    } yield {
+      // merge Import Service and cWDS results
```

Comment:
not for this PR, but maybe for the metrics theme: this might be a good spot to add some metric instrumentation to understand migration status:

- measure latency of each dispatch for comparison
- measure the # of jobs returned from each for comparison